kafka_utils/kafka_check/commands/offline.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Yelp Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import absolute_import
+
+from kafka_utils.kafka_check import status_code
+from kafka_utils.kafka_check.commands.command import KafkaCheckCmd
+from kafka_utils.util.metadata import get_topic_partition_metadata
+
+
+class OfflineCmd(KafkaCheckCmd):
+
+    def build_subparser(self, subparsers):
+        subparser = subparsers.add_parser(
+            'offline',
```

Comment:
It would be, but i wanted to use the same naming convention that we use for the `under_replicated` check, that doesn't have `partitions` in it.